spec/text_spec.rb

```diff
@@ -0,0 +1,22 @@
+require './lib/text'
+
+describe Text do
+  let(:text) {double :text}
+
+  describe 'text' do
+    it 'sends a payment confirmation text message' do
+      allow(text).to receive(:send_text).and_return('text sent')
+      expect(text.send_text).to eq 'text sent'
+    end
+  end
+
+    it 'delivers time' do
+    allow(text).to receive(:delivery_time).and_return 12
+    expect(text.delivery_time).to eq 12
+  end
```

Comment:
Lint/BlockAlignment: `end` at 16, 2 is not aligned with `it 'delivers time' do` at 13, 4.